Python/Product/EnvironmentsList/Resources.resx

```diff
@@ -429,4 +429,18 @@
   <data name="ToolWindow_ListOfEnvironmentsName" xml:space="preserve">
     <value>Python environments</value>
   </data>
+  <data name="NoDBExtensionDisplayName" xml:space="preserve">
+    <value>IntelliSense [database disabled]</value>
```

Comment:
Since this doesn't change after the user re-enables databases, is there a fixed text that we could use that doesn't appear to be wrong? Not a big deal for a temporary UI though.

Reply:
The databases don't reenable immediately anyway. When they restart VS, this item won't be added to the list and the regular "IntelliSense" one will be there.